sisl/geometry.py

```diff
@@ -2264,13 +2265,19 @@ def angle(self, atoms, dir=(1., 0, 0), ref=None, rad=False):
             return ang
         return np.degrees(ang)
 
-    def rotate(self, angle, v, origin=None, atoms: Optional[AtomsArg]=None, only='abc+xyz', rad=False) -> Geometry:
-        """ Rotate geometry around vector and return a new geometry
+    @deprecate_argument("only", "what",
```

Comment:
With this branch installed I didn't get the deprecation message when calling with `only`. Shouldn't I?

Reply:
Thanks! Fixed!